src/plugin/python/PythonRunner.cpp

```diff
@@ -196,21 +196,21 @@ void PythonRunner::runScriptInt(String path, String function, String parameter)
 
 	if (pModule != NULL)
 	{
-		PyObject* pFunc = PyObject_GetAttrString(pModule, function.c_str());
+		PyObject* pFunc = PyObject_GetAttrString(pModule, CSTR(function));
 		/* pFunc is a new reference */
 
 		if (pFunc && PyCallable_Check(pFunc))
 		{
 			PyObject* pArgs = NULL;
 
-			if (parameter.c_str() == NULL)
+			if (CSTR(parameter) == "")
```

Comment:
If you're checking for NULL c_str in UnicodeString it's enough to check if isEmpty() or =="" (it's really the same, but I prefer isEmpty())